Add a known-issues entry for missing attributes (#5515)
elastic/fleet-server: docs/release-notes/known-issues.md

In this {{fleet}}'s logs this will appear as:
```shell
elastic fail 400: document_parsing_exception: [1:209] object mapping for [local_metadata] tried to parse field [local_metadata] as object, but found a concrete value
Eat bulk checkin error; Keep on truckin'
```

and in the {{agent}} it will appear as:
```shell
"log.level":"error","@timestamp":"2025-04-22:12:35:25.295Z","message":"Eat bulk checkin error; Keep on truckin'","component":{"binary":"fleet-server","dataset":"elastic_agent.fleet_server","id":"fleet-server-es-containerhost","type":"fleet-server"},"log":{"source":"fleet-server-es-containerhost"},"service.type":"fleet-server","error.message":"elastic fail 400: document_parsing_exception: [1:209] object mapping for [local_metadata] tried to parse field [local_metadata] as object, but found a concrete value","ecs.version":"1.6.0","service.name":"fleet-server","ecs.version":"1.6.0"
```

**Workaround**
Versions 8.17.11+, 8.18.3+, 8.19.3+, 9.0.3+, and 9.1.0+ include the attribute in the template mapping but do not bump the template's `_meta.managed_index_mappings_version` indicator so the new attribute may not be properly applied.

> Users should upgrade to a version with the template fix and re-enroll an agent.

Author: We're seeing the issue occur mostly with the missing `complete` mapping. Do you have examples for the error messages and other attributes that are caused by the same issue?

> Users should upgrade to a version with the template fix and re-enroll an agent.

Reviewer: We should specify which versions will be fixed so suers don't read this and immediately want to ask that question.

2. After the output confirms all files were successfully processed, run the `enroll` command again.

:::

:::{dropdown} .fleet-agents template is missing mappings

> **Applies to**: {{fleet}} 8.17, 8.18, 8.19, 9.0, 9.1

Reviewer 2: Or based on line 102 should it be this instead?

* 8.17.11+
* 8.18.3 to 8.18.7
* 8.19.3
* 9.0.3 to 9.0.7
* 9.1.0 to 9.1.3

Eat bulk checkin error; Keep on truckin'
```

and in the {{agent}} it will appear as:
```shell
"log.level":"error","@timestamp":"2025-04-22:12:35:25.295Z","message":"Eat bulk checkin error; Keep on truckin'","component":{"binary":"fleet-server","dataset":"elastic_agent.fleet_server","id":"fleet-server-es-containerhost","type":"fleet-server"},"log":{"source":"fleet-server-es-containerhost"},"service.type":"fleet-server","error.message":"elastic fail 400: document_parsing_exception: [1:209] object mapping for [local_metadata] tried to parse field [local_metadata] as object, but found a concrete value","ecs.version":"1.6.0","service.name":"fleet-server","ecs.version":"1.6.0"
```

This attribute was added to the template in versions: 8.17.11 8.18.3, 8.19.3, 9.0.3, 9.1.0.

Further investigation revealed that the `.fleet-agents` index template was not correctly applied due to an unchanged `_meta.managed_index_mappings_version` number.
This change also effects other attributes as well, such as `upgrade_attempts`, `namespaces`, `unprivileged`, and `unhealthy_reason`.

> If there is an error message that indicates any of these attributes it will appear with similar error messages.

Reviewer 2: The current phrasing seems redundant. Maybe:

```suggestion
If there is an error related to any of these attributes, there will be a similar error message in the logs.
```
